Maple2.Model/Metadata/ItemMetadata.cs

```diff
@@ -15,6 +17,12 @@ public record ItemMetadata(
     ItemMetadataMusic? Music,
     ItemMetadataHousing? Housing) : ISearchResult;
 
+public record ItemMetadataLife(
+    int UsePeriod,
+    DateTime ExpirationPeriod,
```

Comment:
It's probably better not to store as `DateTime` for metadata entries since we don't have a conversion layer. (Metadata types are directly used by both DB and Server due to immutability). MySQL `DateTime` is stored using local system timezone which can technically vary, while I've standardized the game layer to use UTC (even though it's technically wrong, at least it's consistent)

tl;dr Let's use long UTC epoch timestamp for now.